google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/async_client.py

```diff
@@ -199,9 +199,7 @@ async def create_instance(
         parent: str = None,
         instance_id: str = None,
         instance: gba_instance.Instance = None,
-        clusters: Sequence[
-            bigtable_instance_admin.CreateInstanceRequest.ClustersEntry
-        ] = None,
+        clusters: Dict[str, gba_instance.Cluster] = None,
```

Comment:
This is inconsistent with the docstring. Are we removing types from docstrings now like we've been in the BigQuery client?

Reply:
Just realized this is generated code. Perhaps the `owlbot` rule could be tweaked to do the same replacement in the docstring?

Is there a bug open for the code generator for this replacement?

Reply:
https://github.com/googleapis/gapic-generator-python/issues/689

Reply:
@tswast f7441f4 fixes up the docstring to match.